docs/REST/throttling.md

```diff
@@ -5,9 +5,13 @@ Configuration stores have limits on the requests that they may serve. Any reques
 Throttling is divided into different quota policies:
 
 **TotalRequests** - total amount of requests/sec
-**TotalWrites** - total amount of write requests/second
+		
+**WriteRequests** - total amount of write requests/second
+
 **Bandwidth** - outbound data in bytes
```

Comment:
Should this be "outbound data in bytes/second"?

Reply:
Good point, but I think it's the opposite. The time period should be omitted from these descriptions. Each of these different policies could be throttled over the span of a second/minute/hour/day

@Yiming-Jia

Reply:
Makes sense. Wording wise, we can say ".... per time unit". But how can a customer figure out it's second, minute or something else?

Reply:
We can document our throttling policies somewhere in our official document(https://azure.microsoft.com/en-us/services/app-configuration). Here we only explain restful api related concepts.

Reply:
@zhenlan from the error object it can't be deduced. @drago-draganov, we may want to surface this information.

Also, I went back and looked and these policy names are outdated. Will work with Yiming on his new PR to make sure that the old names are updated.